service/src/main/java/uk/nhs/adaptors/gp2gp/mhs/InboundMessageConsumer.java

```diff
@@ -1,36 +1,28 @@
-package uk.nhs.adaptors.gp2gp.consumers;
+package uk.nhs.adaptors.gp2gp.mhs;
 
-import com.fasterxml.jackson.databind.ObjectMapper;
 import lombok.RequiredArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.jms.annotation.JmsListener;
 import org.springframework.stereotype.Component;
 
-import uk.nhs.adaptors.gp2gp.models.MhsInboundMessage;
-import uk.nhs.adaptors.gp2gp.services.GP2GPService;
-import uk.nhs.adaptors.gp2gp.utils.JmsReader;
-
 import javax.jms.JMSException;
 import javax.jms.Message;
 import java.io.IOException;
 
 @Component
 @Slf4j
 @RequiredArgsConstructor(onConstructor = @__(@Autowired))
-public class MHSInboundMessageConsumer {
-    private final ObjectMapper objectMapper;
-    private final GP2GPService gp2GPService;
+public class InboundMessageConsumer {
+
```

Comment:
Please remove empty space. We already have an indentation to show that we are working with an inner class code.

Reply:
Minor formatting formatting should be automated so we can focus on more important things during code reviews.